shared/bsp/bsp_can_bridge.h

```diff
@@ -69,7 +71,9 @@ class CanBridge {
   float cooling_limit2 = 0;
   float speed_limit1 = 0;
   float speed_limit2 = 0;
-
+  unsigned int chassis_flag = 0;
+  bool self_check_flag = false;
+  // each bit represents a flag correspond to specific motor e.g.(at index 0, it represents the motor 1's connection flag)
```

Comment:
Very good comment. (It has an example. That's important)
However, the "index 0" of a bitmap might be a little ambiguous because of little endian vs big endian. Consider using "the most significant bit" or "the least significant bit".